rollup.config.js

```diff
@@ -0,0 +1,37 @@
+import typescript from '@rollup/plugin-typescript';
+import { babel } from '@rollup/plugin-babel';
+
+export default {
+	input: {
+		editorial: 'packages/editorial/index.ts',
+		source: 'packages/source/index.ts',
+	},
+	output: {
+		format: 'esm',
+		sourcemap: true,
+		dir: 'dist',
+		entryFileNames: '[name]/index.js',
+	},
```

Comment:
Are we dropping support for CommonJS?

Reply:
do we need to support it? we can add it, i wondered about it. doesn't anyone use it? ought they to be if they are?

Reply:
Providing a CJS endpoint is one of the ways we can get Jest to pick up Source components in consuming applications' test suite. We document this here:

https://theguardian.design/2a1e5182b/p/863906-faqs/t/488531

There might be a better way of doing it (e.g. transpiling a subset of `node_modules` through `babel-jest`).

There were a couple of purely node projects about a year ago that wanted to consume Source, but maybe they've been put out to pasture.